quickview/ui_mock.py

```diff
@@ -0,0 +1,55 @@
+from getter_ui import Ui_Dialog
+from PyQt4 import QtGui
+
+
+class Getter(Ui_Dialog,QtGui.QDialog):
```

Comment:
Class called `Getter` means nothing to me. Needs docstring, but proper fix is to name it something meaningful. Is this a `FileLoaderDialog`?